src/components/Login.js

```diff
@@ -22,6 +22,10 @@ function Login(props) {
 
     let textLabel = context.user ? "Logged in as" : "Enter Your Name";
 
-    return <TextField sx={props.sx} label={textLabel} variant="outlined" onBlur={handleUserNameChange} value={userName || context.user?.id} disabled={!!context.user}/>
+    return context.user ? (
```

Comment:
This is currently hidden _and_ disabled when the user is null; should pick one or the other

Reply:
True, reverted in new commit.